src/components/IdentityView/IdentityView.scss

```diff
@@ -2,11 +2,10 @@
   position: relative;
   padding: $base-padding;
   margin: {
-    bottom: 2*$base-padding;
+    bottom: 2 * $base-padding;
```

Comment:
Question:
Why / Has the prettier config changed?

Reply:
Yes, it did.

Reply:
Question:
As just discussed live, can you please check what's off with this?